core/src/test/java/org/infinispan/persistence/sifs/SoftIndexFileStoreTest.java

```diff
@@ -41,6 +64,7 @@ protected Configuration buildConfig(ConfigurationBuilder configurationBuilder) {
             .addSoftIndexFileStore()
             .dataLocation(Paths.get(tmpDirectory, "data").toString())
             .indexLocation(Paths.get(tmpDirectory, "index").toString())
+            .indexSegments(1)
```

Comment:
This one needed to make the index delay?

Reply:
No, it isn't required, just makes the test easier to diagnose and trace